rero_ils/modules/patrons/api.py

```diff
@@ -307,3 +309,62 @@ def organisation_pid(self):
             patron_type = PatronType.get_record_by_pid(self.patron_type_pid)
             return patron_type.organisation_pid
         return None
+
+    @property
+    def has_valid_subscription(self):
```

Comment:
is it a check to see if he has an active subscription? then possibly `has_an_active_subscription` instead